app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_likes_counter
+    user.increment!(:posts_count)
```

Comment:
- [ ] Kindly make sure to use "increment!" with `author` not `user`, as you set up relations to the author, this method should look like 
```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```